config/routes.rb

```diff
@@ -15,6 +15,8 @@
       only: [:create, :edit, :update]
   end
 
+  get 'my-account', to: 'users#index', as: 'my-account'
+
```

Comment:
Verify, since changing from users#show to users#index, is causing errors in the redirect. 

When creating a new user: 
![imagen](https://user-images.githubusercontent.com/12243331/107594139-d4a9c000-6bd6-11eb-9392-7564a1746c1c.png)

In the edit account window, click Cancel. 
![imagen](https://user-images.githubusercontent.com/12243331/107594148-dbd0ce00-6bd6-11eb-9895-0b0b358e2706.png)
In the edit account window, click Edit. 
![imagen](https://user-images.githubusercontent.com/12243331/107594166-e4290900-6bd6-11eb-9f77-96204c65a7e2.png)

Reply:
Done